static/js/ui.js

```diff
@@ -14,20 +14,24 @@ exports.replace_emoji_with_text = function (element) {
 };
 
 exports.set_up_scrollbar = function (element) {
-    element.perfectScrollbar({
+    var perfectScrollbar = new PerfectScrollbar(element[0], {
         suppressScrollX: true,
         useKeyboard: false,
         wheelSpeed: 0.68,
+        scrollingThreshold: 50,
```

Comment:
From perfect-scroll documentation
"
scrollingThreshold {Number}
This sets threashold for ps--scrolling-x and ps--scrolling-y classes to remain. In the default CSS, they make scrollbars shown regardless of hover state. The unit is millisecond.
Default: 1000
"

I have manually tested and found that the default value of 1000ms does not look well. With a value of 1000ms, if you scroll, stop scrolling, it will take 1 second for the scrollbar to change from darkened gray (user is clicking on scrollbar), back to to regular gray (user may be hovering scrollbar, but not clicking).

The darkened gray scrollbar on click/scrolling does not come with PS's css (its in zulip's components.css), so it makes sense that the default value needs adjustment to be lower. I found 50ms to work well.